src/Samples/BgTaskComponent/README.md

```diff
@@ -2,19 +2,51 @@
 
 This sample demonstrates how to author an out-of-process C#/WinRT component using background tasks, and how to consume the component as a project reference from a packaged .NET 5 WPF app.
 
-To build this sample, set **WpfApp.Package** as the startup project before building the solution. The background task raises a toast notification and is triggered by a time zone change. 
-
 This sample includes the following projects:
 
 - **BgTaskComponent**: This is a C#/WinRT component with an example background task that pops a toast notification.
+
 - **WpfApp** and **WpfApp.Package**: These projects demonstrate hosting the background task component in a packaged .NET 5 desktop (WPF) application.
+
   - **WpfApp** has a project reference to **BgTaskComponent**.
+
   - **WpfApp.Package** is a packaging app with a reference to **WpfApp**. The packaging app is required for hosting out-of-process WinRT components.
 
+## Build and run the sample
+
+### Build the sample
+
+1. Open the **BgTaskComponent.sln** in Visual Studio.
+
+2. Right click on the **WpfApp.Package** project as select **Set as Startup Project**.
+
+3. Press Ctrl+Shift+B, or select **Build** -> **Build Solution**.
+
+### Deploy and run the  sample
+
+1. Select **Build** -> **Deploy Solution** to deploy the sample.
+
+2. To debug and run the sample, press F5 or select **Debug** -> **Start Debugging**.
+
+### Triggering background tasks
+
+The **ToastBgTask** background task raises a toast notification and is triggered by a TimeZoneChange event. You can invoke the background task while running the sample either by using the **Lifecycle Events** dropdown in Visual Studio, or by changing your device's timezone in **Date & time settings**.
```

Comment:
![bg_task_before](https://user-images.githubusercontent.com/70237359/111232100-a76e7a00-85a7-11eb-9f3e-0a269210a1ac.jpg)
![bg_task_after](https://user-images.githubusercontent.com/70237359/111232101-a89fa700-85a7-11eb-87bb-a0a89dc21e78.jpg)

Reply:
these might be nice